spec.bs

```diff
@@ -3631,6 +3631,13 @@ The <dfn for=Navigator method>updateAdInterestGroups()</dfn> method steps are:
           set |ig|'s [=interest group/trusted bidding signals keys=] to |value|.
         1. Otherwise, jump to the step labeled <i><a href=#abort-update>Abort update</a></i>.
 
+        <dt>"`userBiddingSignals`"
+        <dd>
+        1. If |value| is a [=string=],
+          set |ig|'s [=interest group/user bidding signals=] to the result of [=parse a JSON
+          string to an Infra value=] given |value|.

```

Comment:
In discussing with @HabibiYou, we noted that this value is already parsed into infra types, so it can just be stored directly into the IG's userBiddingSignals rather than be reparsed.

Reply:
Has this been resolved yet or is it still waiting on @HabibiYou ?

Reply:
This is resolved in the next commit i made.